build.assets/Makefile

```diff
@@ -381,6 +439,52 @@ enter-root: buildbox
 		-e HOME=$(SRCDIR)/build.assets -w $(SRCDIR) $(BUILDBOX) /bin/bash
 
 #
+# Reconfigure crosstool-NG for the given ARCH via its menuconfig system.
+# e.g.
+# $ make config/buildbox-ng CROSSTOOLNG_ARCH=arm64
+#
+# After saving and exiting in the menuconfig, you should have an updated
+# defconfig file in build.assets/buildbox/ct-ng-configs
+#
+# You need to have a local copy of the configuration container image
+# created with: make buildbox-thirdparty-crosstoolng
+#
+.PHONY: config/buildbox-ng
+config/buildbox-ng:
+	@: $(or $(CROSSTOOLNG_ARCH),$(error CROSSTOOLNG_ARCH variable must be set))
+	@: $(or $(wildcard buildbox/ct-ng-configs/$(CROSSTOOLNG_ARCH).defconfig),\
+		$(error No config for arch $(CROSSTOOLNG_ARCH)))
+	docker run -it --rm \
+		--volume $(shell pwd)/..:/home/teleport \
+		--workdir /home/teleport \
+		buildbox-thirdparty-crosstoolng \
```

Comment:
This part breaks as the `buildbox-thirdparty-crosstoolng` container is tagged `teleportXX` where here we expect `latest` (default Docker behavior).

Reply:
Yeah, sorry, that was a side-effect of the renaming. I've fixed that now.